### PR TITLE
Add Chain icons

### PR DESCRIPTION
**Pull request by vkulinich-cl**

### CODE REVIEW

File: src/api/assetDetails.ts [hunk 1]
```diff
@@ -178,13 +183,13 @@ export const getAssetsDetails = (api: ApiPromise) => async () => {
   }
 
   const assets = rawAssetsData.map(([key, data]) => {
+    const id = key.args[0].toString()
+
     const { symbol = "N/A", decimals } =
-      assetsMeta.find(
-        (assetMeta) => assetMeta.id.toString() === key.args[0].toString(),
-      ) || {}
+      assetsMeta.find((assetMeta) => assetMeta.id.toString() === id) || {}
```
Comment (kolebjak) [inline src/api/assetDetails.ts:189]: It think we should be using descriptive names over variable shadowing. Any thoughts?

File: src/sections/stats/components/PieChart/PieChart.tsx [hunk 2]
```diff
@@ -135,13 +140,14 @@ export const PieChart = <T extends DataEntry>({
...
+            id: "",
           })
         }
       }
 
       return acc
     }, [])
     .sort((a) => (a.symbol !== "rest" ? -1 : 0))
-
+  console.log(slices)
```
Comment (kolebjak) [inline src/sections/stats/components/PieChart/PieChart.tsx:151]:
```suggestion
```

File: src/sections/stats/sections/LRNA/components/Distribution/ChartLabel.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const ChartLabel = () => {
       animate={{ opacity: 1, y: 0 }}
     >
       <div sx={{ flex: "column", align: "center", gap: 6 }}>
-        {getAssetLogo("LRNA")}
+        {<AssetLogo id="1" />}
```
Comment (kolebjak) [inline src/sections/stats/sections/LRNA/components/Distribution/ChartLabel.tsx:20]: Would it be possible to use some kind of map or a constant? So we don't have to use id as a plain "1" string

**Review by kolebjak — CHANGES_REQUESTED** [review]

File: src/api/assetDetails.ts (hunk 1 above)
Comment (vkulinich-cl) [reply src/api/assetDetails.ts:189]: what do you mean by this?

Comment (kolebjak) [reply src/api/assetDetails.ts:189]: In the scope of `find`, `assetMeta` is shadowed by the same variable name which could be considered as an anti-pattern. 

Can be solved by choosing different name for each element, ex: 

```
assetsMeta.find((assetMetaItem) => assetMetaItem.id.toString() === id) || {}
```

Comment (kolebjak) [reply src/api/assetDetails.ts:189]: God dammit, i take it all back :D misread the `s`. Sorry

File: src/components/MultipleIcons/MultipleIcons.styled.ts [hunk 4]
```diff
@@ -21,14 +21,14 @@ export const IconsWrapper = styled.div<{ size: ResponsiveValue<number> }>`
...
-    );
-  }
+  #> :not(:last-of-type) {
+  #  mask: radial-gradient(
+  #    circle at
+  #     calc(var(--circle-size) + var(--mask-radius) + var(--additional-space)),
+  #   transparent calc(var(--mask-radius) + var(--additional-space)),
+  #   white 0
+  # );
+  #}
```
Comment (kolebjak) [inline src/components/MultipleIcons/MultipleIcons.styled.ts:31]:
```suggestion
```

**Review by kolebjak — CHANGES_REQUESTED** [review]

**Review by kolebjak — APPROVED** [review]

**Pull request merged**